src/vimageHosting.h

```diff
@@ -0,0 +1,123 @@
+#ifndef VGITHUBIMAGEHOSTING_H
+#define VGITHUBIMAGEHOSTING_H
+
+#include <QObject>
+#include <QtNetwork>
+#include <QProgressDialog>
+#include <QDesktopServices>
+#include <QUrl>
+#include <QByteArray>
+#include <QNetworkAccessManager>
+#include <QNetworkRequest>
+#include <QNetworkReply>
+#include <QApplication>
+#include <vfile.h>
+#include <QClipboard>
+
+class VGithubImageHosting : public QObject
+{
+    Q_OBJECT
+public:
+    explicit VGithubImageHosting(VFile *p_file, QWidget *parent = 0);
```

Comment:
Parameters should be `p_` prefixed.
```suggestion
    explicit VGithubImageHosting(VFile *p_file, QWidget *p_parent = nullptr);
```